lightning/backend/local.go

```diff
@@ -474,6 +536,7 @@ func (local *local) WriteToTiKV(
 		if count >= local.batchWriteKVPairs || size >= regionMaxSize || totalCount >= regionMaxKeyCount {
 			for i := range clients {
 				requests[i].Chunk.(*sst.WriteRequest_Batch).Batch.Pairs = pairs[:count]
+				log.L().Debug("uploading kvs", zap.Stringer("firstKey", kv.Key(pairs[0].Key)))
```

Comment:
is this log really necessary?

Reply:
No, this is for debugging usage(so we can figure out how many thunks we split this range into), I will remote it then.